components/map/Map.js

```diff
@@ -17,13 +28,15 @@ export default function Map({ users }) {
       scrollWheelZoom={true}
       maxBounds={boundsMap}
       maxBoundsViscosity={0.7}
-      style={{ height: "100vh" }}
+      style={{height:'100%'}}
```

Comment:
The whole problem can be fixed by using `height: 'min(96vw, 100vh)'`

This removes the need for all the JavaScript & global CSS overrides.

This style will also nee to be applied to the wrapper div in the `pages/map.js`

Reply:
This change should be on the MapContainer.
There is no need for a wrapper div.